Makefile

```diff
@@ -54,4 +51,9 @@ run-test-rollback:
 wait-for-docker-postgres:
 	./wait_for_docker_postgres.sh
 
-test-patches: pull-latest-dev-postgres dev-postgres-pre-down dev-postgres-up wait-for-docker-postgres run-test-patches run-test-rollback dev-postgres-down
+test-patches: pull-latest-dev-postgres dev-postgres-down dev-postgres-up wait-for-docker-postgres run-test-patches run-test-rollback dev-postgres-down
+
+build-dev-postgres-image:
+	docker build ./dev-common-postgres-image -t europe-west2-docker.pkg.dev/ssdc-rm-ci/docker/ssdc-rm-dev-common-postgres:latest
+
+test-dev-postgres: build-dev-postgres-image dev-postgres-up wait-for-docker-postgres run-test-patches run-test-rollback dev-postgres-down
```

Comment:
I don't think this needs to run the test patches or rollback, neither will see any changes to run and removing those would probably let you get rid of the Python setup from the workflow to trim it down even more?